controllers/rollingupgrade_controller.go

```diff
@@ -458,9 +465,9 @@ func (r *RollingUpgradeReconciler) getNodeFromAsg(i *autoscaling.Instance, nodeL
 	return nil
 }
 
-func (r *RollingUpgradeReconciler) populateAsg(ruObj *upgrademgrv1alpha1.RollingUpgrade) error {
+func (r *RollingUpgradeReconciler) populateAsg(ruObj *upgrademgrv1alpha1.RollingUpgrade, force bool) error {
```

Comment:
Please remove this change. We added the populateAsg() at multiple cases to handle the edge cases. This means, we have to cache at that point in time else we see failures in upgrades. So, all the  calls for populateAsg() will end up having a ` true`. 
So, for now just call a populateAsg() as soon as we put the instance to standby. 

PS: There is a bigger refactor already in place to handle such in situations in upgrade-manager-v2